### PR TITLE
merge js-ipfs ipfs-message-port-protocol@0.4.2

### PR DESCRIPTION
**Pull request by laipogo**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```
Comment (taiyangc) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Remove issue template

File: .github/workflows/bundlesize.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Checks bundlesize, does not run on master
+on:
+  pull_request:
+    branches:
+      - '*'
+
+name: Bundlesize
```
Comment (taiyangc) [inline .github/workflows/bundlesize.yml:7]: May need to update something to get workflows to work? Or just get rid of it if hard to modify?

File: .idea/workspace.xml [hunk 3]
```diff
@@ -0,0 +1,1861 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (taiyangc) [inline .idea/workspace.xml:1]: remove?

File: Dockerfile.latest [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+RUN npm install --unsafe-perm -g ipfs@"$IPFS_VERSION"
+
+# Make the image a bit smaller
+RUN npm cache clear --force
+RUN apk del build-base python git
+
+# Configure jsipfs
+RUN jsipfs init
+
+RUN jsipfs version
```
Comment (taiyangc) [inline Dockerfile.latest:19]: Update?

File: README.md [hunk 5]
```diff
@@ -1,458 +1,186 @@
...
-  <a href="https://app.fossa.io/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fipfs%2Fjs-ipfs-http-client?ref=badge_small" alt="FOSSA Status"><img src="https://app.fossa.io/api/projects/git%2Bhttps%3A%2F%2Fgithub.com%2Fipfs%2Fjs-ipfs-http-client.svg?type=small"/></a>
-  <br>
-  <a href="https://github.com/feross/standard"><img src="https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat-square"></a>
-  <a href="https://github.com/RichardLitt/standard-readme"><img src="https://img.shields.io/badge/standard--readme-OK-green.svg?style=flat-square" /></a>
-  <a href=""><img src="https://img.shields.io/badge/npm-%3E%3D3.0.0-orange.svg?style=flat-square" /></a>
-  <a href=""><img src="https://img.shields.io/badge/Node.js-%3E%3D10.0.0-orange.svg?style=flat-square" /></a>
-  <br>
+  <a href="https://js.ipfs.io" title="JS IPFS">
+    <img src="https://ipfs.io/ipfs/Qme6KJdKcp85TYbLxuLV7oQzMiLremD7HMoXLZEmgo6Rnh/js-ipfs-sticker.png" alt="IPFS in JavaScript logo" width="244" />
+  </a>
```
Comment (taiyangc) [inline README.md:4]: Need to revert to btfs lines

**Review by taiyangc — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (laipogo) [reply .github/ISSUE_TEMPLATE/config.yml:1]: rm .github directory

File: .github/workflows/bundlesize.yml (hunk 2 above)
Comment (laipogo) [reply .github/workflows/bundlesize.yml:7]: okay, just get rid of it

**Pull request closed without merging**